config/config.exs

```diff
@@ -36,7 +36,8 @@ config :livebook,
   plugs: [],
   rewrite_on: [],
   shutdown_callback: nil,
-  teams_auth?: false,
+  teams_auth: nil,
+  teams_auth_hub_id: nil,
```

Comment:
I would store `teams_auth: {:online | :offline, id} | nil`, assuming we actually need the online/offline info (if we set the provider only for online upfront).

Reply:
But we already have this from `:teams_auth`. Do we need to put it again?

Reply:
My point is that instead of having `teams_auth` and `teams_auth_hub_id`, we could have both in a single config.

Reply:
Still valid, if I would store a single field. But provided we set `:identity_provider` for online only, we no longer need to store :online/:offline in the first place.

Reply:
I think we can get rid of the teams_auth_hub_id. So I am closing this.